blazor/datagrid/columns.md

```diff
@@ -1216,4 +1216,59 @@ The following sample code demonstrates [DisplayAsCheckBox](https://help.syncfusi
 The following image represents the DisplayAsCheckBox enabled for a DataGrid column,
 ![Displaying CheckBox in Blazor DataGrid Column](./images/blazor-datagrid-column-with-checkbox.png)
 
+## Stacked column header
+
+The Grid columns can be stacked or grouped in order to show multiple levels of column headers. This can be achieved by nesting the [GridColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn) directive within another `GridColumn` directive.
+
+In this following sample, the columns **Order Date**, **Freight** are grouped under **Order Details** and the columns **Shipped Date**, **Ship Country** are grouped under **Shipped Details**.
```

Comment:
In the following sample, the columns **Order Date**, and **Freight** are grouped under **Order Details** and the columns **Shipped Date**, and **Ship Country** are grouped under **Shipped Details**.